README.md

```diff
@@ -2911,6 +2912,51 @@ Value (string):
   }
 }
 ```
+### GoToIntranetSiteForSingleWordEntryInAddressBar
+Whether to always go through the DNS server before sending a single word search string to a search engine.
+
+If the site exists, it will navigate to the website. If the intranet responds with a 404, the page will show a 404. If the intranet does not respond, the browser will either attempt to search the term.
+
+The second result in the URL bar will be a search result to allow users to conduct a web search exactly as it was entered.
+
+If instead you would like to enable the ability to have your domain appear as a valid URL and to disallow the browser from ever searching that term using the first result that matches it, add the pref `browser.fixup.domainwhitelist.YOUR_DOMAIN` (where `YOUR_DOMAIN` is the name of the domain you'd like to add), and set the pref to `true`. The URL bar will then suggest `YOUR_DOMAIN` when the user fully types `YOUR_DOMAIN`. If the user attempts to load that domain and it fails to load, it will show an "Unable to connect" error page.
+
+You can also whitelist a domain suffix that is not part of the [Public Suffix List](https://publicsuffix.org/) by adding the pref  `browser.fixup.domainsuffixwhitelist.YOUR_DOMAIN_SUFFIX` with a value of `true`.
```

Comment:
I think I added an extra unnecessary white space between "pref  `browser.fixup.domainsuffixwhitelist.YOUR_DOMAIN_SUFFIX`". We can remove that.